### ISSUE TITLE
vagrant-cloud post-processor fails: cannot unmarshal string into Go value of type uint

### ISSUE THREAD

**Issue by pedersen** [mitchellh/packer#1976]
I've been building a base box for TurboGears, and made pretty good progress on it. I now want to make that base box accessible to the public via the vagrant-cloud processor, and it's failing when parsing the json response. I've reached my current limits of debugging this issue, so am attaching the output of the post-processor with PACKER_LOG=1 having been set up.

Version information:
Packer 0.7.5
OS: Ubuntu 14.10, all patches applied

```
2015/03/11 23:25:38 ui: ==> virtualbox-iso: Running post-processor: vagrant-cloud
==> virtualbox-iso: Running post-processor: vagrant-cloud
2015/03/11 23:25:38 ui: ==> virtualbox-iso (vagrant-cloud): Verifying box is accessible: pedersen/tg2pg9
2015/03/11 23:25:38 packer-post-processor-vagrant-cloud: 2015/03/11 23:25:38 Post-Processor Vagrant Cloud API GET: https://vagrantcloud.com/api/v1/box/pedersen/tg2pg9?access_token=ACCESS_TOKEN
==> virtualbox-iso (vagrant-cloud): Verifying box is accessible: pedersen/tg2pg9
2015/03/11 23:25:39 packer-post-processor-vagrant-cloud: 2015/03/11 23:25:39 Post-Processor Vagrant Cloud API Response:
2015/03/11 23:25:39 packer-post-processor-vagrant-cloud: 
2015/03/11 23:25:39 packer-post-processor-vagrant-cloud: &{200 OK %!s(int=200) HTTP/1.1 %!s(int=1) %!s(int=1) map[Content-Type:[application/json; charset=utf-8] Date:[Thu, 12 Mar 2015 03:25:38 GMT] X-Content-Type-Options:[nosniff] Content-Length:[1962] Connection:[keep-alive] Etag:["9aa7c0dbcbde97cf696468809af61c0e"] X-Powered-By:[Phusion Passenger 4.0.59] X-Runtime:[0.483728] X-Xss-Protection:[1; mode=block] Cache-Control:[max-age=0, private, must-revalidate] Status:[200 OK] X-Request-Id:[0b19f234-eb06-42fe-b0e3-86c9b9ee04d4] Server:[Apache/2.2.22 (Ubuntu) Phusion_Passenger/4.0.59] X-Frame-Options:[SAMEORIGIN]] %!s(*http.bodyEOFSignal=&{0xc2080b6200 {0 0} false ?reflect.Value? 0x524a40 0x5249d0}) %!s(int64=1962) [] %!s(bool=false) map[] %!s(*http.Request=&{GET 0xc20807f7a0 HTTP/1.1 1 1 map[Content-Type:[application/json]] <nil> 0 [] false vagrantcloud.com map[] map[] <nil> map[]   <nil>}) %!s(*tls.ConnectionState=&{771 true false 49199  true  [0xc208018900 0xc208019680 0xc208081b00] [[0xc208018900 0xc208019680 0xc208139680] [0xc208018900 0xc208019680 0xc208081b00 0xc2081c5680]]})}
2015/03/11 23:25:39 Deleting original artifact for build 'virtualbox-iso'
2015/03/11 23:25:39 ui error: Build 'virtualbox-iso' errored: 1 error(s) occurred:

* Post-processor failed: Error parsing box response: json: cannot unmarshal string into Go value of type uint
2015/03/11 23:25:39 Builds completed. Waiting on interrupt barrier...
Build 'virtualbox-iso' errored: 1 error(s) occurred:

* Post-processor failed: Error parsing box response: json: cannot unmarshal string into Go value of type uint
2015/03/11 23:25:39 machine readable: error-count []string{"1"}
2015/03/11 23:25:39 ui error: 
==> Some builds didn't complete successfully and had errors:
2015/03/11 23:25:39 machine readable: virtualbox-iso,error []string{"1 error(s) occurred:\n\n* Post-processor failed: Error parsing box response: json: cannot unmarshal string into Go value of type uint"}
2015/03/11 23:25:39 ui error: --> virtualbox-iso: 1 error(s) occurred:

* Post-processor failed: Error parsing box response: json: cannot unmarshal string into Go value of type uint
2015/03/11 23:25:39 ui: 
==> Builds finished but no artifacts were created.
```

**Comment by pedersen** [comment]
And I forgot to add the post processor configuration. The access token is stored in a variable file, and is included via the command line.

```
	    {
		"type": "vagrant-cloud",
		"box_tag": "pedersen/tg2pg9",
		"keep_input_artifact": true,
		"access_token": "{{user `cloud_token`}}",
		"version": "2.3.4.1",
		"version_description": "This release provides support for automatically running a setup.py script, creating the postgresql database, and running the 'gearbox setup_app' commands when running 'vagrant up'."
	    }
```

**Comment by pedersen** [comment]
Have just discovered that this is a known issue in the 0.7.5 release with the vagrant-cloud post-processor. See issue #1735 for the issue this is a duplicate of. I was able to clone and build a development version of packer, and it worked for us.

Closing this out, and apologies for the duplicate.
